src/actions/home/index.cr

```diff
@@ -1,5 +1,11 @@
 class Home::Index < BrowserAction
   get "/" do
-    render LuckyWeb::WelcomePage
+    wod = WodQuery.new.date(Time.now.at_beginning_of_day).first?
+
+    if wod.is_a?(Wod)
+      render Workouts::NewPage, wod: wod
```

Comment:
I think instead of the Home::Index rendering this page directly, it should redirect to a Workouts::New or Workouts::Show. The reason for that is that you might want to change the "Home" at some point. Maybe it is a dashboard, or a list of previous workout with a button to add a new one, etc. By redirecting you give yourself some flexibility